src/schema/users.ts

```diff
@@ -765,9 +765,11 @@ export const resolvers: IResolvers<any, Context> = {
           : user.image;
 
       try {
-        const result = await ctx.con
-          .getRepository(User)
-          .save({ ...user, ...data, image: avatar });
+        const updatedUser = { ...user, ...data, image: avatar };
+        if (!user.infoConfirmed && updatedUser.email && updatedUser.username) {
```

Comment:
```suggestion
        if (!user.infoConfirmed && updatedUser.email && updatedUser.username && updatedUser.name) {
```

Reply:
This one is always required in our auth so it will always have a value, so I did not include it. But I agree, being strict is always safer.